source/user-manual/reference/ossec-conf/global.rst

```diff
@@ -256,17 +256,21 @@ This sets the memory size for the event correlation engine.
 white_list
 ^^^^^^^^^^
 
-This specifies an IP for which Active Responses will not be triggered. Only one IP may be specified for each ``<while_list>`` tag, but several IPs may be used by including multiple ``<white_list>`` tags.
+This specifies an IP, netblock, or hostname for which Active Responses will not be triggered. Only one of those values may
```

Comment:
```suggestion
This specifies an IPv4 Address, netblock, IPv4-mapped IPv6 address or hostname for which Active Responses will not be triggered. Only one of those values may
```

Reply:
Done in [commit](https://github.com/wazuh/wazuh-documentation/pull/3987/commits/3925bf788514d8943e6f9aed35b1152cca46b3d5)